mpas_analysis/test/test_mpas_xarray.py

```diff
@@ -123,7 +123,17 @@ def test_selvals(self):
             ['time_avg_avgValueWithinOceanLayerRegion_avgLayerTemperature',
              'refBottomDepth']
 
-        selvals = {'nVertLevels': 0}
+        vertIndex = 10
```

Comment:
Also, why the departure from the choice of `0`?  This would be the top layer and would be a common use case.